src/components/GoogleAddressLookup/component.js

```diff
@@ -0,0 +1,297 @@
+                    disabled={disabled}
+                    value={getFormattedValue(value, false, <SelectedLocationIcon />)}
+                    onChange={this.handleChange}
+                    onSearch={this.handleSearch}
+                    onClick={onClick}
+                    onFocus={onFocus}
+                    onBlur={onBlur}
+                    error={error}
+                    icon={<LocationIcon />}
+                />
+                {!error && (
+                    <div className="rainbow-google-address-lookup_poweredby-container">
+                        <PoweredByGoogleLogo className="rainbow-google-address-lookup_poweredby-logo" />
+                    </div>
+                )}
+            </div>
+        );
+    }
+}
+
+PlacesLookupComponent.propTypes = {
+    isScriptLoaded: PropTypes.bool.isRequired,
+    isScriptLoadSucceed: PropTypes.bool.isRequired,
+    searchOptions: PropTypes.shape({
+        bounds: PropTypes.shape({
+            sw: PropTypes.shape({
+                latitude: PropTypes.number.isRequired,
+                longitude: PropTypes.number.isRequired,
+            }),
+            ne: PropTypes.shape({
+                latitude: PropTypes.number.isRequired,
+                longitude: PropTypes.number.isRequired,
+            }),
+        }),
+        country: PropTypes.string,
+        location: PropTypes.shape({
+            latitude: PropTypes.number.isRequired,
+            longitude: PropTypes.number.isRequired,
+        }),
+        radius: PropTypes.number,
+        types: PropTypes.arrayOf(PropTypes.string),
+    }),
+    label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+    hideLabel: PropTypes.bool,
+    value: PropTypes.oneOfType([
+        PropTypes.shape({
+            description: PropTypes.string,
+            id: PropTypes.string,
+            matched_substrings: PropTypes.arrayOf(
+                PropTypes.shape({
+                    length: PropTypes.number,
+                    offset: PropTypes.number,
+                }),
+            ),
+            place_id: PropTypes.string,
+            reference: PropTypes.string,
+            structured_formatting: PropTypes.arrayOf(
+                PropTypes.shape({
+                    main_text: PropTypes.string,
+                    main_text_matched_substrings: PropTypes.arrayOf(
+                        PropTypes.shape({
+                            length: PropTypes.number,
+                            offset: PropTypes.number,
+                        }),
+                    ),
+                    secondary_text: PropTypes.string,
+                }),
+            ),
+            terms: PropTypes.arrayOf(
+                PropTypes.shape({
+                    offset: PropTypes.number,
+                    value: PropTypes.string,
+                }),
+            ),
+            types: PropTypes.arrayOf(PropTypes.string),
+        }),
+        PropTypes.string,
+    ]),
```

Comment:
we can make a proptypes.js file where we can export value and searchOptions prop types since are to large and used two times, and here use something like:
```
value: valueShape,
searchOptions: searchOptionsShape,

```